ichnaea/data/area.py

```diff
@@ -102,24 +113,6 @@ def update_area(self, session, areaid):
             return
 
         # Otherwise update the area entry based on all the cells
-        area = session.execute(
```

Comment:
This query fetched the existing cellarea data, but was only used to determine if it already existed (use ``UPDATE``) or should be created (use ``INSERT``).